### PR TITLE
Design vault

### LINKED ISSUE #371: [Add] Design Vault
### URL

https://designvault.io/

### Description of the above provided site

Looking for inspiration for your next project? Unleash your creativity by exploring an ever-growing collection of designs and common user interface patterns from the world’s best products and designers.

### Category

Frontend

### Sub Category

design inspiration

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

Closes #371

## Changes proposed

design-vault added

## Screenshots
Screenshot (780)
Screenshot (781)

## Note to reviewers

give me progressive feedback

### CODE REVIEW

**Review by rupali-codes — COMMENTED** [review]
Hey Parma, could you please update your code and add the specific changes which you have made, in your PR it's also showing the changes of your previous PR which has been already merged, thanks

File: database/frontend/design-inspirations.json [hunk 1]
```diff
@@ -130,5 +130,20 @@
...
+
+    },
+    {
+        "name": "Design Vault",
+        "description": "Design Vault is a website that showcases designs from real products, giving you access to the latest and greatest in UI patterns and trends.",
+        "url": "https://designvault.io/",
+        "category": "frontend",
+        "subcategory": "design-inspiration"
+     }
+]
```
Comment (CBID2) [inline database/frontend/design-inspirations.json:149]:
```suggestion
}
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @parma1111! 😄  I gave you a suggestion on how to solve @rupali-codes' request. I hope it helps! 👍🏾

File: database/frontend/design-inspirations.json (hunk 1 above)
Comment (parma1111) [reply database/frontend/design-inspirations.json:149]: > 

ok I have put curly bracket on line 149
now you can review it

**Review by CBID2 — APPROVED** [review]
LGTM! 👍🏾

**Comment by parma1111** [comment]
> Hey, please put the props into curly braces like other ones

done, in vscode it is not showing any error on deployment but in github pr it is showing error in deployment.
how to resolve it

**Comment by parma1111** [comment]
> > Hey, please put the props into curly braces like other ones
> 
> done, in vscode it is not showing any error on deployment but in github pr it is showing error in deployment. how to resolve it

now it's ok, everything is working

**Review by CBID2 — APPROVED** [review]
Yay you did it @parma1111! 😊 LGTM! 👍🏾

**Comment by parma1111** [comment]
> Yay you did it @parma1111! 😊 LGTM! 👍🏾

Thank you CBID2

**Comment by CBID2** [comment]
> > Yay you did it @parma1111! 😊 LGTM! 👍🏾
> 
> Thank you CBID2

Anytime!

**Review by rupali-codes — COMMENTED** [review]
Please do NOT update or modify package.json and package-lock.json files unless it's necessary. Add the only files you have made changes in, by using this command `git add <file>`

**Comment by parma1111** [comment]
> Please do NOT update or modify package.json and package-lock.json files unless it's necessary. Add the only files you have made changes in, by using this command `git add <file>`

now review the pr

**Review by rupali-codes — COMMENTED** [review]
Could you please reverse package-lock.json as well?

**Comment by parma1111** [comment]
> Could you please reverse package-lock.json as well?

now review the pr

**Comment by rupali-codes** [comment]
> > Could you please reverse package-lock.json as well?
> 
> now review the pr

Thank you, It would be great if you can describe how u did that? :)

**Review by rupali-codes — APPROVED** [review]
looks perfect

**Pull request merged**